src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java

```diff
@@ -224,7 +224,7 @@ public void shouldCorrectlyHandleEmptyValuesInMatchingDataset() {
 
         JSONObject attributes = jsonResponse.getJSONObject("attributes");
         assertThat(attributes.keys()).containsExactlyInAnyOrder(COMMON_ATTRIBUTES);
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
         String expectedFromDateString = standardFromDate.toLocalDate().atStartOfDay().format(formatter).replace(" ", "T");
```

Comment:
We don't need these line string replacement lines anymore.

Reply:
Good point.  I've updated the PR (via an amend/force push, because I also noticed I'd forgotten to put the Trello link in the original commit message).